ixmp4/core/run.py

```diff
@@ -80,6 +61,27 @@ def unset_as_default(self):
 
 
 class RunRepository(BaseFacade):
+    def create(
+        self,
+        model: str,
+        scenario: str,
+    ):
```

Comment:
We can type hint this as 
```suggestion
    ) -> Run:
```